kfs-ld/src/main/java/edu/arizona/kfs/module/ld/LaborConstants.java

```diff
@@ -25,4 +25,7 @@
 	
 	public static final String[] BALANCE_TYPES = {"AC", "CB", "IE"};
 
+	//SalaryExpenseTransferDocument
+	public static final String BALANCE_TYPE_ACTUAL = "AC";
+	public static final String BALANCE_TYPE_A2 = "A2";
```

Comment:
You should really be using System Options to get balance type codes as these could change by the fiscal year (OK, UA values have never changed). So, for the future, this is how it should be done:

`//get current year system options
SystemOptions options = getSystemOptions(universityDateService.getCurrentFiscalYear());

// set to AC
e.setFinancialBalanceTypeCode(options.getActualFinancialBalanceTypeCd());

public SystemOptions getSystemOptions(Integer universityFiscalYear) {
    SystemOptions options = null;
    options = getOptionsService().getOptions(universityFiscalYear);
    if (ObjectUtils.isNull(options)) {
        options = getOptionsService().getCurrentYearOptions();
    } 
    return options;
}`